Cargo.toml

```diff
@@ -34,27 +34,25 @@ tower = { version = "0.4", optional = true }
 axum = { version = "0.7.5", optional = true }
 pin-project-lite = { version = "0.2", optional = true }
 futures-util = { version = "0.3", default-features = false, features = [], optional = true }
-hyper = { version = "0.14", default-features = false, features = ["http1", "client"] }
-hyper-v1 = { package = "hyper", version = "1.3", default-features = false, features = ["http1", "client"], optional = true }
+hyper = { package = "hyper", version = "1.3", default-features = false, features = ["http1", "client"] }
 http-body-util = { version = "0.1.1", optional = true }
 aws-types =  { version = "1", optional = true }
 lambda_runtime = { version = "0", optional = true }
 paste = { version = "1.0.14", optional = true }
-opentelemetry-resource-detectors = "0.2.0"
+hyper-util = "0.1.9"
 
 [dev-dependencies]
 assert2 = "0.3"
 rstest = "0.22"
-tokio = "1.35.1"
+tokio = "1.38"
```

Comment:
For some reason `opentelemetry-http` demands `tokio@1.40` and doesn't work with any other version:

```
error: failed to select a version for `tokio`.
    ... required by package `opentelemetry-http v0.25.0`
    ... which satisfies dependency `opentelemetry-http = "^0.25"` of package `telemetry-rust v3.2.0 (https://github.com/nentgroup/telemetry-rust.git?rev=7c6498e14fb75fdea6a3a95083a3ef4fbbc3f663#7c6498e1)`
    ... which satisfies git dependency `telemetry-rust` (locked to 3.2.0) of package `profiles v1.0.0 (/Users/leonbesc/Documents/viaplay/profiles/profiles)`
versions that meet the requirements `~1.38.0` are: 1.38.1, 1.38.0

all possible versions conflict with previously selected packages.

  previously selected package `tokio v1.40.0`
    ... which satisfies dependency `tokio = "^1.40.0"` of package `profiles v1.0.0 (/Users/leonbesc/Documents/viaplay/profiles/profiles)`

failed to select a version for `tokio` which could resolve this conflict
```